doc/source/index.rst

```diff
@@ -0,0 +1,333 @@
+  :ref:`gatherer` and concatenate files together (it will need `Kai`_ to do
+  so).  When done, it sends another message.
+* For pre-processing data with AAPP and ANA, `aapp-runner`_ is responsible
+  and can be configured to read posttroll messages either from :ref:`cat` or
+  directly from :ref:`gatherer`.  See documentation for `aapp-runner`_.
+
+The exact configuration depends on factors that will vary depending on
+what satellite data are processed, whether thore are from direct readout,
+EUMETCAST, or another source, what system is used for direct readout,
+and other factors.
+Some users use the 3rd party software `supervisor`_ to start and monitor
+the different scripts in pytroll-collectors.
+
+There are example configurations in the ``examples/`` directory.
+
+.. _posttroll: https://posttroll.readthedocs.io/en/latest/
+.. _AAPP: https://nwp-saf.eumetsat.int/site/software/aapp/
+.. _Satpy: https://satpy.readthedocs.io/
+.. _aapp-runner: https://github.com/pytroll/pytroll-aapp-runner
+.. _Kai: https://navigator.eumetsat.int/product/EO:EUM:SW:METOP:165
+.. _supervisor: http://supervisord.org/
+
+Scripts
+-------
+
+All scripts use posttroll messages to communicate.  They are normally
+running in the background (as a daemon).  Most are waiting for a posttroll
+message to trigger their role and will send another posttroll message when
+their task is done.
+
+.. _cat:
+
+cat
+^^^
+
+Concatenates granules or segments to a single file.  This may be a useful step
+before using external software for preprocessing data.  You will need `Kai`_
+from EUMETSAT to concatenate MetOp AVHRR granules.  Cat listens to input
+messages via posttroll according to topics defined in the configuration file.
+Upon completion, it will publish a posttroll message with topic defined in
+the configuration file.
+
+The kai configuration file is in the INI format and should have one ore
+more sections.  Each section may have the following fields:
+
+output_file_pattern
+    A pattern (trollsift syntax) on what the output file should be.
+
+aliases
+    Optional (what does it do?)
+
+min_length
+    Optional, integer, minimum number of minutes needed to consider the data
+
+command
+    Command used for concatenation.
+
+stdout
+    Optional; if command writes to stdout, redirect output here.
+
+publish_topic
+    Optional; publish a message when file is produced, using this topic.
+
+publish_port
+    Optional; use a custom port when publishing a message.
+
+nameservers
+    Optional; nameservers to publish on.
+
+subscriber_nameserver
+    Optional; nameserver to listen to.
+
+.. _Kai: https://navigator.eumetsat.int/product/EO:EUM:SW:METOP:165
+
+Example configuration:
+
+.. literalinclude:: ../../examples/cat.cfg_template
+   :language: ini
+
+catter
+^^^^^^
+
+Alternative to cat that does something else.
+
+.. literalinclude:: ../../examples/catter.cfg_template
+   :language: ini
+
+.. _gatherer:
+
+gatherer
```

Comment:
- collects granulated swath data so that the granules cover the configured target area(s) in a contiguous manner
- uses `pytroll-schedule` (which uses `pyorbital`) to calculate the required granules using orbital parameters (TLE)

Reply:
Added, thanks.